chain/client/src/stateless_validation/state_witness_producer.rs

```diff
@@ -45,14 +45,16 @@ impl Client {
             .ordered_chunk_validators();
 
         let my_signer = self.validator_signer.as_ref().ok_or(Error::NotAValidator)?.clone();
+        let witness_size;
```

Comment:
this would be more idiomatic: 
```
let (witness, witness_size) = {
  ...
  (ChunkStateWitness { inner: witness_inner, signature }, witness_size)
}
```